code/schemes-numbers.md

```diff
@@ -1,15 +1,15 @@
-| CRI scheme number | URI scheme | Reference   |
```

Comment:
In Section 8.1 "Converting Between CoAP CRIs and Sets of CoAP Options", the second column in Table 1 is currently "URI scheme". That should also be "URI scheme name", for consistency with the present Table 6 of Appendix A and with the text in Section 8.1.